ste/folderCreationTracker.go

```diff
@@ -55,7 +55,7 @@ func (f *nullFolderTracker) StopTracking(folder string) {
 type jpptFolderTracker struct {
 	plan                   IJobPartPlanHeader
 	mu                     *sync.Mutex
-	contents               map[string]uint32
+	contents               *common.Trie
 	unregisteredButCreated map[string]struct{}
```

Comment:
I think we should be able to also contain the unregistered but created info in the same Trie right?

Reply:
Addressed. Please review